tf2_ros/include/tf2_ros/async_buffer_interface.h

```diff
@@ -39,11 +40,44 @@
 #include <functional>
 #include <future>
 #include <string>
+#include <utility>
 
 namespace tf2_ros
 {
 
-using TransformStampedFuture = std::shared_future<geometry_msgs::msg::TransformStamped>;
+class TransformStampedFuture : public std::shared_future<geometry_msgs::msg::TransformStamped>
+{
+  typedef std::shared_future<geometry_msgs::msg::TransformStamped> _Base_type;
```

Comment:
Nit. Just to maintain consistency with other `using` directives in this patch.
```suggestion
  using _Base_type = std::shared_future<geometry_msgs::msg::TransformStamped>;
```